src/Util/Test.php

```diff
@@ -517,7 +518,7 @@ private static function getDataFromDataProviderAnnotation($docComment, $classNam
                     $data = $dataProviderMethod->invoke($object, $methodName);
                 }
 
-                if ($data instanceof Iterator) {
+                if ($data instanceof Traversable) {
                     $data = \iterator_to_array($data);
```

Comment:
@keradus You're right, but as @issei-m has mentioned - `iterator_to_array` expects `\Traversable`.